### PR TITLE
Add emission_weight

### PR DESCRIPTION
**Pull request by portsmouth**
Implements the change described in https://github.com/AcademySoftwareFoundation/OpenPBR/issues/229.

image

image

### CODE REVIEW

**Review by ZapAndersson — APPROVED** [review]
I approve.

File: index.html [hunk 1]
```diff
@@ -1128,17 +1128,18 @@
...
+Being an intensity, **`emission_luminance`** can be any value greater than or equal to zero. For convenience, we make the soft range $[0, 1000]$, corresponding to the typical range of home appliances. (Note that if the renderer does not deal with photometric units internally, a scale factor may need to be applied to bring the emission into a sensible range).
 
 
 Emission params          | Label     | Type     | Range           | Norm          | Default       | Description
 -------------------------|-----------|----------|:---------------:|:-------------:|:-------------:|----------------------------------------------
-**`emission_luminance`** | Luminance | `float`  | $ [0, \infty) $ | $ [0, 1000] $ | $ 0         $ | Emission luminance, in cd/m^2 (aka. nits)
-**`emission_color`**     | Color     | `color3` | $ [0, 1]^3    $ |               | $ (1, 1, 1) $ | Emission color multiplier
+**`emission_weight`**    | Weight    | `float`  | $ [0, 1]      $ |               | $ 0          $ | Emission weight luminance multiplier
```
Comment (virtualzavie) [inline index.html]: Slight adjustment of the table alignement:

```suggestion
**`emission_weight`**    | Weight    | `float`  | $ [0, 1]      $ |               | $ 0         $ | Emission weight luminance multiplier
```

**Comment by portsmouth** [comment]
~~`emission_weight` luminance~~ `emission_luminance` needs to default to 1000 in both the text and the parametrization table.

**Comment by virtualzavie** [comment]
> `emission_weight` luminance needs to default to 1000 in both the text and the parametrization table.

Ah right, I see the parameter list still has a luminance default value of 0 (assuming you meant `emission_luminance`, not `emission_weight`).
If we have two parameters, indeed, one of them must have a default value different from 0.

The default value 1000 sounds reasonable to me since it's the order of magnitude of the maximum luminance of typical displays screens (smartphones and TVs).

**Comment by jstone-lucasfilm** [comment]
This looks fine to me, though it would be ideal to update the reference implementation in sync with this change, so that we don't accidentally create divergence between them.

**Comment by portsmouth** [comment]
>This looks fine to me, though it would be ideal to update the reference implementation in sync with this change, so that we don't accidentally create divergence between them.

Done in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/231/commits/eed50221cb278fb03eb30d1ae9937ad52044000a

**Comment by portsmouth** [comment]
@jstone-lucasfilm can be merged?

**Comment by jstone-lucasfilm** [comment]
@portsmouth This looks very close to the mark, but we should update Anton's `open_pbr_lightbulb.mtlx` example so that it remains an emissive material in OpenPBR 1.2!

https://github.com/AcademySoftwareFoundation/OpenPBR/blob/main/examples/open_pbr_lightbulb.mtlx

**Comment by portsmouth** [comment]
Done in 203103e749e4a8991cd529513bbbf6c9808d0779.  Hopefully there aren't a lot of existing assets out there that will be bitten by this (I assume not, yet..).

**Comment by portsmouth** [comment]
> @portsmouth This looks very close to the mark, but we should update Anton's `open_pbr_lightbulb.mtlx` example so that it remains an emissive material in OpenPBR 1.2!
> 
> https://github.com/AcademySoftwareFoundation/OpenPBR/blob/main/examples/open_pbr_lightbulb.mtlx

Done in 203103e749e4a8991cd529513bbbf6c9808d0779.  Hopefully there aren't a lot of existing assets out there that will be bitten by this (I assume not, yet..).

**Review by jstone-lucasfilm — APPROVED** [review]
This looks good to me, thanks @portsmouth!

**Pull request merged**